src/components/webView/iframe.native.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { View } from 'react-native'
+import { Text } from 'react-native'
+
+// const wHeight = getMaxDeviceHeight()
+
+export const createIframe = (src, title) => {
```

Comment:
lets use react-native-webview

Reply:
The objective its resolve crash, but i can introduce react-native-webview on other Pr because this is not included in react-native like before, its a library apart.